packages/itwinui-react/src/core/Carousel/CarouselSlider.tsx

```diff
@@ -86,7 +86,7 @@ export const CarouselSlider = React.forwardRef((props, ref) => {
 
   return (
     <Box
-      as='ol'
+      as='div'
```

Comment:
Might not be needed because `<Box>` default is `div`, but other than that LGTM

Reply:
You were right! I removed `div` and it still works perfectly